level/level_test.go

```diff
@@ -233,3 +272,122 @@ func TestInjector(t *testing.T) {
 		t.Errorf("wrong level value: got %#v, want %#v", got, want)
 	}
 }
+
+func TestParse(t *testing.T) {
+	testCases := []struct {
+		name    string
+		level   string
+		want    level.Value
+		wantErr bool
```

Comment:
Please change this to an `error` type and check the specific error value returned in the test.